src/encoded/schemas/long_read_rna_quality_metric.json

```diff
@@ -0,0 +1,151 @@
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run", "award", "lab", "replicates_sequencing_depth", "mapping_rate", "replicates_genes_detected"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+   ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "replicates_sequencing_depth": {
+            "title": "Replicates sequencing depth",
+            "description": "Number of full-length non-chimeric reads per isogenic replicate",
+            "type":  "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "Replicate read count",
+                "description": "Full-length non-chimeric read counts per replicate",
+                "type": "object",
+                "required": ["replicate", "full_length_non_chimeric_read_count"],
+                "properties": {
+                    "replicate": {
+                        "title": "Replicate",
+                        "description": "Isogenic replicate",
+                        "type": "string",
+                        "linkTo": "Replicate"
+                    },
+                    "full_length_non_chimeric_read_count": {
+                        "title": "Read count",
+                        "description": "Quantity of full-length reads not mapping chimerically of the genome",
+                        "type": "integer",
+                        "minimum": 0
+                    }
+                }
+            }
+        },
+        "replicates_mapping_rate": {
+            "title": "Replicates mapping rate",
+            "description": "Percentage of reads mapping to reference genome per isogenic replicate",
+            "type":  "array",
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "title": "Replicate mapping rate",
+                "description": "Mapping rate per isogenic replicate",
+                "type": "object",
+                "required": ["replicate", "mapping_rate"],
+                "properties": {
+                    "replicate": {
+                        "title": "Replicate",
+                        "description": "Isogenic replicate",
+                        "type": "string",
+                        "linkTo": "Replicate"
+                    },
+                    "mapping_rate": {
+                        "title": "Mapping rate",
+                        "type":  "number",
```

Comment:
I think the mapping rate should be a float between 0 and 1.0

Reply:
adressed